scania-digital-design-system/sdds
# Improvement/dropdown styling moved

**Describe pull-request**  
Dropdown styling variables moved from the theme package(`_dropdowns.scss`) to the component package

**Solving issue**  

Fixes: [AB#1501](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1501)

**How to test**  
1. Go to Storybook example -> Dropdown component
2. Check the Dropdown look and behaviour are same as current one

**Additional context**  
the spacing is still from theme package, `theme/spacing/spacing-vars.scss`
